37515 Firefox Nightly supports Temporal behind pref (#37863)
mdn/content: files/en-us/mozilla/firefox/experimental_features/index.md

  </tbody>
</table>

### Temporal API

> The [Temporal object](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal) enables date and time management in various scenarios, including built-in time zone and calendar representation. ([Firefox bug 1912511](https://bugzil.la/1912511)). This includes:

Reviewer: ```suggestion
The [Temporal object](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal) aims to simplify working with dates and times in various scenarios, with built-in time zone and calendar representations. ([Firefox bug 1912511](https://bugzil.la/1912511)). This includes:
```

  </tbody>
</table>

### Temporal API

> The [Temporal object](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal) aims to simplify working with dates and times in various scenarios, with built-in time zone and calendar representations. ([Firefox bug 1912511](https://bugzil.la/1912511)). This includes:

Reviewer: ```suggestion
The [Temporal object](/en-US/docs/Web/JavaScript/Reference/Global_Objects/Temporal) aims to simplify working with dates and times in various scenarios, with built-in time zone and calendar representations ([Firefox bug 1912511](https://bugzil.la/1912511)).
This includes:
```

- A **duration** (difference between two time points): {{jsxref("Temporal.Duration")}}
- **Points in time**:
  - As a unique instant in history:
    - A timestamp: {{jsxref("Temporal.Instant")}}
    - A date-time with a time zone: {{jsxref("Temporal.ZonedDateTime")}}
  - **Time-zone-unaware date/time ("Plain")**:
    - Date (year, month, day) + time (hour, minute, second, millisecond, nanosecond): {{jsxref("Temporal.PlainDateTime")}}
      - Date (year, month, day): {{jsxref("Temporal.PlainDate")}}
        - Year, month: {{jsxref("Temporal.PlainYearMonth")}}
        - Month, day: {{jsxref("Temporal.PlainMonthDay")}}
      - Time (hour, minute, second, millisecond, nanosecond): {{jsxref("Temporal.PlainTime")}}

> - The current time as various class instances, or in a specific format: {{jsxref("Temporal.Now")}}

Reviewer: ```suggestion
- **Now** (current time) as various class instances, or in a specific format: {{jsxref("Temporal.Now")}}
```

<table>
  <thead>
    <tr>
      <th>Release channel</th>
      <th>Version added</th>
      <th>Enabled by default?</th>
    </tr>
  </thead>
  <tbody>
    <tr>
      <th>Nightly</th>
      <td>135</td>
      <td>No</td>
    </tr>
    <tr>
      <th>Developer Edition</th>
      <td>—</td>
      <td>—</td>
    </tr>
    <tr>
      <th>Beta</th>
      <td>—</td>
      <td>—</td>
    </tr>
    <tr>
      <th>Release</th>
      <td>—</td>
      <td>—</td>
    </tr>
    <tr>
      <th>Preference name</th>

>       <td colspan="2">javascript.options.experimental.temporal</td>

Reviewer: ```suggestion
      <td colspan="2"><code>javascript.options.experimental.temporal</code></td>
```
